pallets/vc-management/src/lib.rs

```diff
@@ -175,6 +192,10 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
+		/// a delegatee doesn't exist
+		DelegateeNotExist,
```

Comment:
:nail_care: the grammar in the error name feels a bit broken tbh, but it's already broken in the other error names so maybe it's better to keep it consistent? :sweat_smile:

Reply:
Yes it's intentionally written in this way to be consistent with other pallets 😂 

but maybe it's better to look through similar errors in substrate and adjust them all in our code

Reply:
I'll leave it to another small issue